pages/linux/dex.md

```diff
@@ -0,0 +1,32 @@
+# dex
+
+> DesktopEntry Execution is a program to generate and execute DesktopEntry files of the Application type.
+> More information: <https://github.com/jceb/dex>.
+
+- Perform an autostart/execute all programs in the autostart folders:
+
+`dex --autostart`
+
+- Perform an autostart/execute all programs in the specified folders:
+
+`dex --autostart --search-paths {{path/to/directory}}:{{path/to/directory}}`
+
+- Preview the programs would be executed in a regular autostart:
+
+`dex --autostart --dry-run`
+
+- Preview the programs would be executed in a GNOME specific autostart:
+
+`dex --autostart --dry-run --environment GNOME`
```

Comment:
```suggestion
`dex --autostart --dry-run --environment {{GNOME}}`
```

Reply:
@Waples why the downvote?  This value is variable, thus, according to our guidlines, this should be done.

Reply:
GNOME is an environment, make it `{{environment}}` or `{{desktop_environment}}`. 
Change to `{{GNOME}}` makes no sense.

Reply:
Same as https://github.com/tldr-pages/tldr/pull/5752#discussion_r614307425

Reply:
As a matter of fact, frequently we have examples like this:
```
- Do such and such a thing with a value of `abc`:

`command --value {{abc}}`
```
So, even though we state the value we're demonstrating in the description itself, we still tokenize it because it _is_ in fact variable.